server/tinylicious/src/app.ts

```diff
@@ -78,6 +85,22 @@ export function create(config: Provider, storage: IDocumentStorage, mongoManager
 		next(err);
 	});
 
+	// API to broadcast a signal to a container
+	app.use(
+		Router().post("/broadcast-signal", (req, res) => {
+			try {
+				const tenantId = req.body?.tenantId;
+				const doucmentId = req.body?.doucmentId;
+				const signalContent = req.body?.singalContent;
+				const payload : IBroadcastSignalEventPayload = { tenantId, doucmentId, signalContent };
+				collaborationSessionEventsEmitter.emit("broadcast-signal", payload);
```

Comment:
Can the request body ever be undefined here? And in that case would you want to still emit that payload?